src/server/JasmineGraphServer.cpp

```diff
@@ -1381,44 +1251,45 @@ static bool removePartitionThroughService(string host, int port, string graphID,
     }
 
     if (!Utils::sendExpectResponse(sockfd, data, INSTANCE_DATA_LENGTH, partitionID, JasmineGraphInstanceProtocol::OK)) {
+        Utils::send_str_wrapper(sockfd, JasmineGraphInstanceProtocol::CLOSE);
         close(sockfd);
         return false;
     }
 
+    Utils::send_str_wrapper(sockfd, JasmineGraphInstanceProtocol::CLOSE);
     close(sockfd);
     return true;
 }
 
-std::vector<JasmineGraphServer::workers> JasmineGraphServer::getHostWorkerMap() { return hostWorkerMap; }
-
 void JasmineGraphServer::updateOperationalGraphList() {
     string hosts = "";
     string graphIDs = "";
     std::vector<std::string> hostsList;
 
-    if (profile == Conts::PROFILE_NATIVE) {
+    if (jasminegraph_profile == PROFILE_NATIVE) {
         hostsList = Utils::getHostListFromProperties();
-    } else if (profile == "docker") {
+    } else if (jasminegraph_profile == PROFILE_DOCKER) {
         hostsList = getWorkerVector(workerHosts);
+    } else {
+        return;  // TODO(thevindu-w): implement.for k8s
```

Comment:
Can you give more details here in the comment? What is missing to be implemented for k8s?

Reply:
It should get the list of worker addresses. But I'm not sure if address alone is sufficient to uniquely identify it. Otherwise, it will become a bug.

Reply:
Ok, @Ishad-M-I-M what's you opinion on this?

Reply:
For K8s mode the IP address itself is sufficient for uniquely identify a worker. Not like in docker mode where it uses the ip address + port combination

Reply:
Ok